juriscraper/opinions/united_states/state/neb.py

```diff
@@ -11,33 +11,29 @@ def __init__(self, *args, **kwargs):
         self.status = "Published"
 
     def _process_html(self):
-        path = {
-            "group": '//div[contains(@class, "view-grouping")]',
-            "date": './/span[contains(@class, "date-display-single")]',
-            "row": ".//tbody/tr",
-        }
-        for group in self.html.xpath(path["group"]):
-            date_element = group.xpath(path["date"])
-            if not date_element:
+        for table in self.html.xpath(".//table"):
+            date_tags = table.xpath("preceding::time[1]/text()")
+            if not date_tags:
+                print("NO DATE TAGS")
```

Comment:
Should use a logger call (logger.error if you want it to go to Sentry, logger.warning / info for normal use)

Reply:
this is the one thing im not sure about - I think it should be fine without it ... because we will get the error if no date exists.